packages/build-utils/package.json

```diff
@@ -41,7 +41,7 @@
     "execa": "3.2.0",
     "fs-extra": "10.0.0",
     "glob": "8.0.3",
-    "ignore": "4.0.6",
+    "ignore": "5.2.4",
```

Comment:
I used the existing `getIgnoreFilter` from `build-utils`, but that function does not work as written. It uses the newer `5.0.0+` interface, but the dependency was pinned to a lower version for some reason.